src/ActionModal/components/ActionModal.vue

```diff
@@ -18,6 +18,16 @@
           <i class="material-icons notranslate">close</i>
         </div>
         <span class="action-modal-title">{{ title }}</span>
+        <TmFormMsg
```

Comment:
put in the respective action modals: DelegationModal/UndelegationModal. Please read the acceptance criteria in #4017